Makefile

```diff
@@ -29,6 +29,10 @@ deploy-dev: manifests
 	kubectl apply -f config/crds
 	kustomize build config/overlays/development | kubectl apply -f -
 
+deploy-test:
```

Comment:
deploy-dev?

Reply:
You are right I named `deploy-test` since Clive already created one. @cliveseldon what do you think if we can just replace your `deploy-dev` with what I am doing in `deploy-test` :) ?